frontend/src/app/modules/work_packages/routing/wp-full-view/wp-full-view.component.ts

```diff
@@ -34,6 +34,7 @@ import {Component, Injector, OnInit} from '@angular/core';
 import {WorkPackageViewSelectionService} from 'core-app/modules/work_packages/routing/wp-view-base/view-services/wp-view-selection.service';
 import {WorkPackageSingleViewBase} from "core-app/modules/work_packages/routing/wp-view-base/work-package-single-view.base";
 import {BackRoutingService} from "core-app/modules/common/back-routing/back-routing.service";
+import {of} from "rxjs";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/081ba59455a4ea64fa1f7c7f1e0a8120f02c3ee4/frontend/src/app/modules/work_packages/routing/wp-full-view/wp-full-view.component.ts#L38"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>